GestorDeEstoque/Controllers/UsuarioController.cs

```diff
@@ -0,0 +1,44 @@
+using GestorDeEstoque.Data;
+using GestorDeEstoque.DTOs;
+using GestorDeEstoque.Models;
+using GestorDeEstoque.Repositories;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GestorDeEstoque.Controllers
+{
+    [ApiController]
+    [Route("/usuario")]
```

Comment:
O padrão na rota normalmente é plural

```suggestion
    [Route("/usuarios")]
```